element_zstack/volume.py

```diff
@@ -104,8 +116,9 @@ class Volume(dj.Imported):
     """
 
     def make(self, key):
+        """Populate the Volume table with volumetric microscopic imaging data."""
         vol_tif_fp = get_volume_tif_file(key)
-        volume_data = TiffFile(vol_tif_fp).asarray()
+        volume_data = TiffFile(vol_tif_fp[0]).asarray()
```

Comment:
```suggestion
        volume_data = TiffFile(vol_tif_fp).asarray()
```

Reply:
From the description above of `get_volume_tif_file` it is supposed to return a single file.